h/static/styles/group-form.scss

```diff
@@ -50,6 +49,12 @@
       margin-bottom: 40px;
     }
 
+    &__heading-short {
```

Comment:
If you introduce a variation of an element, the convention is to use '--modifier-name', so `&__heading--short` in this case.